src/Components/GameOver.tsx

```diff
@@ -37,11 +38,12 @@ const GameOver = () => {
   const {gameDispatch} = useGame();
   const gameOver = useGameOverDocState();
   const gameWon = useGameWonDocState();
+  const { setMusicOn, soundOn, playCheeringSound, playLoseSound } = useAudio();
 
   useEffect(() => {
-    if (gameOver && gameWon) playCheering();
-    else if (gameOver && !gameWon) playLose();
-  }, [gameOver, gameWon])
+    if (gameOver && gameWon && soundOn) playCheeringSound() && setMusicOn(false) ;
```

Comment:
actually can you help refactor these to
```
if (!soundOn) return;
if (!gameOver) return;

if (gameWon) playCheeringSound();
else playLoseSound();
```

Also i dont quite understand why you are doing setMusicOn(false)

Reply:
I did `setMusicOn(false)` because the music still plays when the cheering sound is on. I want it to stop playing once the game is won or lost

```
  useEffect(() => {
    if (!soundOn) return;
    if (!gameOver) return;

    if (gameWon) playCheeringSound() && setMusicOn(false) ;
    else playLoseSound() && setMusicOn(false);
  }, [gameOver, gameWon, soundOn])